src/react-apps/applications/runtime/src/features/form/containers/index.tsx

```diff
@@ -38,14 +38,14 @@ export default (props) => {
       `${window.location.origin}/${org}/${service}/api/resource/FormLayout.json`,
     );
     FormDataActions.fetchFormData(
-      `${window.location.origin}/${org}/${service}/api/${partyId}/Index/${instanceId}`,
+      `${window.location.origin}/${org}/${service}/api/${partyId}/${instanceId}`,
     );
     FormRuleActions.fetchRuleModel(
       `${window.location.origin}/${org}/${service}/api/resource/RuleHandler.js`,
     );
     FormWorkflowActions.getCurrentState(
       // tslint:disable-next-line:max-line-length
-      `${window.location.origin}/${org}/${service}/api/workflow/${partyId}/GetCurrentState?instanceId=${instanceId}`,
+      `${window.location.origin}/${org}/${service}/api/workflow/${partyId}/${instanceId}/GetCurrentState`,
```

Comment:
Do we need the partyId here if it is included in the instanceId.. 
Ref the update you made to src/react-apps/applications/runtime/src/features/instantiate/containers/index.tsx

Reply:
I've tested that the get GetCurrentState works in Studio (test).